components/NinjaForm/index.js

```diff
@@ -44,22 +45,22 @@ export default function NinjaForm({ id }) {
           setUserSkills(response.data);
           setSelectedSkills(response.data.map((s) => s.id));
         })
-        .catch((error) => notification["error"](error.data?.errors));
+        .catch(notifyError);
     }
   }, [id]);
 
   const deleteSkill = (ninja_id, skill_id) => {
     api
       .deleteNinjaSkills(ninja_id, skill_id)
       .then((_) => getUserSkills())
-      .catch((error) => notification["error"](error.data?.errors));
+      .catch(notifyError);
   };
 
   const addSkill = (ninja_id, skill_id) => {
     api
       .addNinjaSkills(id, skill_id)
       .then((_) => getUserSkills())
-      .catch((error) => notification["error"](error.data?.errors));
+      .catch(notifyError);
```

Comment:
For example here, if you can't add a skill, it is most likely because the user already has that skill.